api/authenticate.go

```diff
@@ -24,6 +24,34 @@ import (
 // Authenticate represents the API handler to
 // process a user logging in to Vela from
 // the API or UI.
+// swagger:operation GET /authenticate authenticate Authenticate
```

Comment:
Thoughts on moving the function description comments below the swagger specific comments?

My thought process is that it might help (or be nice) to remain consistent 👍

Reply:
To clarify - do something similar to login.go?
https://github.com/go-vela/server/blob/4a8063d440152c6a580ff9466a68bd400489bc9d/api/login.go#L77-L78

Reply:
@meinenec Yes that is correct 👍

Reply:
Good catch on these :)

Reply:
✋ 5️⃣  - there is no :teamwork: or :high_five: emoji's that I can find 😞